.github/workflows/aws-terraform-integration-tests.yml

```diff
@@ -32,12 +48,16 @@ jobs:
           architecture: ${{ matrix.architecture }}
           distribution: ${{ matrix.distribution }}
 
-      - run: java -version
+      - name: JDK info
+        run: java -version
 
       # BUILD HAZELCAST AWS SNAPSHOT
-      - uses: actions/checkout@v3.3.0
+      - name: Checkout
+        uses: actions/checkout@v3.3.0
         with:
           path: hazelcast
+          ref: ${{steps.decide-ref.outputs.ref}}
```

Comment:
We do not have a decide-ref step here if I am not wrong, could you provide it?

Reply:
You mean use only `ref: ${{ github.ref }}`

Reply:
No, like this https://github.com/hazelcast/hazelcast-platform-operator/blob/main/.github/workflows/pull-request.yml#L220-L224